src/main/java/org/opentripplanner/routing/algorithm/raptoradapter/transit/DefaultAccessEgress.java

```diff
@@ -0,0 +1,63 @@
+package org.opentripplanner.routing.algorithm.raptoradapter.transit;
+
+import org.opentripplanner.routing.algorithm.raptoradapter.transit.cost.RaptorCostConverter;
+import org.opentripplanner.routing.core.State;
+import org.opentripplanner.transit.raptor.api.transit.RaptorAccessEgress;
+import org.opentripplanner.util.lang.ToStringBuilder;
+
+/**
+ * Default implementation of the RaptorAccessEgress interface.
+ */
+public class DefaultAccessEgress implements RaptorAccessEgress {
```

Comment:
Should we rename this back to `AccessEgress` after the interface was renamed to `AccessEgress`

Reply:
No really strong opinion about it but I would think that when the IDE autocompletion lists "RaptorAccessEgress" and "DefaultAccessEgress", it is clear which one is the interface and which one is the implementation. It is less obvious with "RaptorAccessEgress" and "AccessEgress".